src/libxtp/jobcalculators/qmmm.h

```diff
@@ -47,13 +47,18 @@ class QMMM final : public ParallelXJobCalc<std::vector<Job> > {
 
  private:
   bool hasQMRegion() const;
+  Job createJob(const Segment& seg, const QMState& state, Index jobid);
```

Comment:
```suggestion
  Job createJob(const Segment& seg, const QMState& state, Index jobid)const;
```
Could we make it `const` ?